packages/vsce/src/utils/profileManagement.ts

```diff
@@ -342,19 +305,9 @@ export class ProfileManagement {
         return toArray(response.records.cicsmanagedregion);
       }
     } catch (error) {
-      let errorMessage = `Error requesting CICSManagedRegion - ${JSON.stringify(error)}`;
-      if (error instanceof CicsCmciRestError) {
-        if (error.RESPONSE_1 === CicsCmciConstants.RESPONSE_1_CODES.NOTAVAILABLE) {
```

Comment:
How are we getting away with no longer checking for a `NOTAVAILABLE` resp1 code that doesn't fail?